src/osvr/Client/TrackerRemoteFactory.cpp

```diff
@@ -142,21 +142,12 @@ namespace client {
 
         shared_ptr<RemoteHandler> ret;
 
-        /// @todo set this struct correctly from the descriptor, or perhaps
-        /// the path?
+        /// @todo set this struct correctly from the descriptor
         VRPNTrackerHandler::Options opts;
 
-        auto interfaceType = source.getInterfaceName();
-        if ("position" == interfaceType) {
-            opts.reportPosition = true;
-        } else if ("orientation" == interfaceType) {
-            opts.reportOrientation = true;
-        } else {
-            /// pose and tracker both imply full reports (?)
-            opts.reportOrientation = true;
-            opts.reportPosition = true;
-            opts.reportPose = true;
-        }
+        opts.reportOrientation = true;
+        opts.reportPosition = true;
+        opts.reportPose = true;
```

Comment:
Yes. This was the only code path that was ever actually getting called anyway, which is why it was confusing. There was no way to get into any other path but the `tracker` one without making a really weird device descriptor.